templates/include/telemetry/environment.1.schema.json

```
@@ -656,6 +656,16 @@
                     "string",
                     "null"
                   ]
+                },
+                "type": {
+                  "type": [
+                    "string",
+                    "null"
+                  ],
+                  "enum": [
+                    "SSD",
+                    "HDD"
```

```suggestion
                    "HDD",
                    null
```

This should allow null values for this field, while also limiting the values to the enumerated set.

---

PR updated!